pystan/model.py

```diff
@@ -754,6 +755,9 @@ def sampling(self, data=None, pars=None, chains=4, iter=2000,
         fit.stan_args = args_list
         fit.stanmodel = self
         fit.date = datetime.datetime.now()
+
+        throwaway = pystan.diagnostics.check_MCMC_diagnostics(fit)
```

Comment:
Should this be `verbose=True` or always on?

No need to name the call. 
Add a comment that says that this will print diagnostics (and if we decide the verbose thing then if `verbose=True`)

Reply:
I was planning on having the diagnostics be always on, since RStan does the same thing.